ios/wrappers/utils/CioConfigUtils.swift

```diff
@@ -0,0 +1,58 @@
+import CioInternalCommon
+import CioDataPipelines
+import CioMessagingInApp
+
+extension Region: Decodable {}
+
+let logLevels = Set<String>(CioLogLevel.allCases.map(\.rawValue))
+
+struct QASettings: Decodable {
+    let cdnHost: String
+    let apiHost: String
+}
+
+struct RCTCioConfig: Decodable {
+    let cdpApiKey: String
+    let siteId: String?
+    let region: Region?
+    let trackApplicationLifecycleEvents: Bool?
+    let enableInApp: Bool?
+    let qa: QASettings?
+    
+    static func from(_ json: AnyObject) throws -> Self {
+        let data = try JSONSerialization.data(withJSONObject: json, options: [])
+        let instance = try JSONDecoder().decode(Self.self, from: data)
+        return instance
+    }
+}
+
+struct CioConfig {
```

Comment:
Will this be exposed to public? If yes, I would suggest to use full name instead i.e. `CustomerIOConfig`

Reply:
yes, this will be exposed to public. This is same as `CustomerIOConfig` as we have today, just a rename and has been called out in the [tech proposal](https://www.notion.so/custio/Tech-Plan-React-native-as-DataPipeline-Source-f8544804df3f4c4bb9b0d4a4d70d7fad?pvs=4#15086f3e00e3464f82cb94e2510ae6d3) as well. 

I’m okay with renaming it back to CustomerIOConfig if there’s a strong reason to do so!
For me, `CioConfig` is short and precisely describes the context such as within a module or codebase whenever we use “CIO” it is commonly understood to refer to Customer.io.